README.md

```diff
@@ -60,50 +141,132 @@ repository at the time of the pull.
+
+This is a large patch which updates multiple portions of the codebase.
+```
+$ cd $HOME/zopen/dev/v8base/v8
+$ git apply ../../v8port/patches/git.20231122.fce38915e4.diff
 ```
-cd v8
-git apply ~/v8port/patches/git.20231122.fce38915e4.diff
 
-pushd build
-git apply ~/v8port/patches/build/git.20231122.968682938b.diff
-popd
+This is a small patch which updates the toolchain config.
+```
+$ cd $HOME/zopen/dev/v8base/v8
+$ pushd build
+$ git apply ../../../v8port/patches/build/git.20231122.968682938b.diff
+$ popd
+```
+From this point forward, `pushd/popd` is used so that the directory
+is not specifed each time.
 
-pushd third_party/abseil-cpp
-git apply ~/v8port/patches/third_party/abseil-cpp/git.20231122.1e8861f03f.diff
-popd
+This is a small patch which updates the third-party abseil common c++ libaries.
+```
+$ pushd third_party/abseil-cpp
+$ git apply ../../../../v8port/patches/third_party/abseil-cpp/git.20231122.1e8861f03f.diff
+$ popd
+```
 
-pushd third_party/googletest/src
-git apply ~/v8port/patches/third_party/googletest/src/git.20231122.af29db7ec2.diff
-popd
+This is a tiny patch which adds a new ZOS platform to google test.
 ```
-# Files specific to z/OS only
+$ pushd third_party/googletest/src
+$ git apply ../../../../../v8port/patches/third_party/googletest/src/git.20231122.af29db7ec2.diff
+$ popd
+```
+
+
+##### PAX Workflow
+
 Extract the files from the .pax files, by running `pax -p p -rzf <pax-file>`,
 using the same approach. Example:
 
 ```
-cd v8
-pax -p p -rzf ~/v8port/patches/src.zos.20231122.fce38915e4.pax
+$ cd $HOME/zopen/dev/v8base/v8
+$ pax -p p -rzf ../../v8port/patches/src.zos.20231122.fce38915e4.pax
 
-pushd buildtools
-pax -p p -rzf ~/v8port/patches/buildtools/buildtools.zos.20231122.92b79f4d75.pax
-popd
+$ pushd buildtools
+$ pax -p p -rzf ../../../v8port/patches/buildtools/buildtools.zos.20231122.92b79f4d75.pax
+$ popd
 ```
-# Download, build and install zoslib:
+
+
+##### Download, build and install zoslib:
+
+
 ```
-pushd v8/third_party
-git clone git@github.com:ibmruntimes/zoslib && cd zoslib
-./build.sh -c -r -t
-popd
+$ cd $HOME/zopen/dev/v8base/v8
+$ pushd third_party
+$ git clone git@github.com:ibmruntimes/zoslib 
+$ cd zoslib
+$ ./build.sh -c -r -t
+$ popd
 ```
+
+Output:
+
+```
+$ ./build.sh -c -r -t
+~/zopen/dev/v8base/v8/third_party/zoslib/build ~/zopen/dev/v8base/v8/third_party/zoslib
+-- The CXX compiler identification is unknown
+-- The C compiler identification is unknown
+-- The ASM compiler identification is GNU
+-- Found assembler: /bin/xlclang
+-- Detecting CXX compiler ABI info
+-- Detecting CXX compiler ABI info - failed
+-- Check for working CXX compiler: /bin/xlclang++
+-- Check for working CXX compiler: /bin/xlclang++ - broken
+'[31mCMake Error at /z/jd895801/zopen/usr/local/zopen/cmake/CMake-heads.v3.27.8.20231124_184234.zos/share/cmake-3.27/Modules/CMakeTestCXXCompiler.cmake:60 (message):
+  The C++ compiler
+
+    "/bin/xlclang++"
+
+  is not able to compile a simple test program.
```

Comment:
It seems your xlclang is not correctly setup. Also, V8 builds with clang/clang++, so for all work in V8, including build of zoslib, you need to have;
```
export CC=clang
export CXX=clang++
export LINK=$CXX
```

Reply:
Yes, that is the ticket.  I didn't setup the compiler.  Perhaps in the install this was mentioned as something to do by users.  Thanks for the clarification though. I appreciate it.